portfolio/src/main/webapp/photo_gallery.html

```diff
@@ -0,0 +1,107 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Photo Gallery</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <style>
+    body{
+        background-color: #a8d0e6;
+    }
+    div.gallery {
+        margin: 5px;
+        border: 3px solid;
+        float: left;
+        width: 200px;
+    }
+    div.gallery img {
+        width: 100%;
+        height: auto;
+    }
+    div.img {
+        padding: 15px;
+        text-align: center;
+    }
+</style>
+<body>
+
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_3112.JPG">
+    <img src="/images/IMG_3112.JPG" alt="Cinque Terre" width="600" height="400">
+  </a>
+  <div class="img">GHC 2019</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_3053.JPG">
+    <img src="/images/IMG_3053.JPG" alt="Northern Lights" width="600" height="400">
+  </a>
+  <div class="img">"I see myself at Google"</div>
+</div>
+
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_3052.JPG">
+    <img src="/images/IMG_3052.JPG" alt="Mountains" width="600" height="400">
+  </a>
+  <div class="img">The big "G"</div>
+</div>
+<div class="gallery">
+  <a target="_blank" href="/images/IMG_2897.JPG">
+    <img src="/images/IMG_2897.JPG" alt="a" width="600" height="400">
+  </a>
+  <div class="img">First time surfing in San Diego</div>
+</div><div class="gallery">
+  <a target="_blank" href="/images/IMG_2264.JPG">
+    <img src="/images/IMG_2264.JPG" alt="b" width="600" height="400">
+  </a>
+  <div class="img">Chicago Bean</div>
+</div><div class="gallery">
+  <a target="_blank" href="/images/IMG_1796.JPG">
+    <img src="/images/IMG_1796.JPG" alt="c" width="600" height="400">
+  </a>
+  <div class="img">H-town</div>
+</div><div class="gallery">
+  <a target="_blank" href="/images/IMG_1255.JPG">
+    <img src="/images/IMG_1255.JPG" alt="s" width="600" height="400">
+  </a>
+  <div class="img">Universal Studios</div>
+</div><div class="gallery">
+  <a target="_blank" href="/images/IMG_4803.jpg">
+    <img src="/images/IMG_4803.jpg" alt="d" width="600" height="400">
+  </a>
+  <div class="img">Graduation</div>
+</div><div class="gallery">
+  <a target="_blank" href="/images/IMG_4664.JPG">
+    <img src="/images/IMG_4664.JPG" alt="f" width="600" height="400">
+  </a>
+  <div class="img">Prom</div>
+</div><div class="gallery">
+  <a target="_blank" href="/images/IMG-4217.JPG">
+    <img src="/images/IMG-4217.JPG" alt="g" width="600" height="400">
+  </a>
+  <div class="img">Buckingham Palace</div>
+</div><div class="gallery">
+  <a target="_blank" href="/images/IMG-0678.JPG">
+    <img src="/images/IMG-0678.JPG" alt="h" width="600" height="400">
+  </a>
+  <div class="img">Starbucks Reserve</div>
+</div>
+<div class="gallery">
+  <a target="_blank" href="/images/IMG-4274.jpg">
+    <img src="/images/IMG-4274.jpg" alt="j" width="600" height="400">
```

Comment:
I think you need to "git add" all these other images as well to add them to this pull request